modules/apps/frontend-taglib/frontend-taglib-clay/build.gradle

```diff
@@ -1,11 +1,23 @@
 dependencies {
+	compileInclude group: "com.fasterxml.jackson.core", name: "jackson-annotations", version: "2.10.3"
+	compileInclude group: "com.fasterxml.jackson.core", name: "jackson-core", version: "2.10.3"
+	compileInclude group: "com.fasterxml.jackson.core", name: "jackson-databind", version: "2.10.3"
```

Comment:
@brianchandotcom Regarding your comment on https://github.com/brianchandotcom/liferay-portal/pull/90147#issuecomment-644931995 I checked and it doesn't compile without those.